test/multidevice.h

```diff
@@ -99,6 +99,7 @@ class PipelineTest : public MultiDeviceTest {
   std::vector<c10::IValue> inputs;
   std::vector<c10::IValue> unsharded_inputs;
   std::vector<at::Tensor> outputs;
+  bool auto_schedule_in_execute = false;
```

Comment:
Does `_in_execute` bear a special meaning?

Reply:
Originally, for distinguishing with "_in_validate", but I will remove it